_includes/about/_leadership.html

```diff
@@ -57,8 +57,8 @@ <h1 class="component-header-box text-tan">Leadership</h1>
       </div>
 
       <div class="col-md-4 push-bottom">
-        <crds-portrait-card theme="expanded" text-color="white" href="/media/authors/kyle-ranson" lead="Teaching Pastor/Director of ET" title="Kyle Ranson" bg-overlay="true" image-src="//crds-media.imgix.net/2LgGIDcVfk1mFMnz4X2ts2/d44d87888e6e82d6b9dc3b12f12b7f7c/leadership-kyle-ranson.png?h=250">
-          <p>Kyle is the Director of the Experience Team at Crossroads, they create stuff like videos, music, articles, and weekend services. He joyfully fulfills stereotypes about Milennials with his love of craft beer, tight jeans, and woodworking, and is passionate about people finding God.</p>
+        <crds-portrait-card theme="expanded" text-color="white" href="/media/authors/kyle-ranson" lead="Online 'Anywhere' Church Pastor" title="Kyle Ranson" bg-overlay="true" image-src="//crds-media.imgix.net/2LgGIDcVfk1mFMnz4X2ts2/d44d87888e6e82d6b9dc3b12f12b7f7c/leadership-kyle-ranson.png?h=250">
```

Comment:
For both references of **Online 'Anywhere' Church Pastor** I think we should keep it as the **"Online "Anywhere" Church Pastor"** from the task. I believe you can do `lead='Online "Anywhere" Church Pastor'` if the quotes are an issue.